resources/RegexGrammar.pp

```diff
@@ -42,11 +42,13 @@
 //
 
 // Character classes.
+%token  negative_class_fc_       \[\^(?=\])         -> class_fc
+%token  class_fc_                \[(?=\])           -> class_fc
```

Comment:
Is the indent intentional different in a few of this lines?

Reply:
it looks weird, but I tried to follow the original indentation